src/main/java/knightminer/simplytea/data/gen/RecipeGenerator.java

```diff
@@ -94,6 +96,18 @@ protected void buildRecipes(Consumer<FinishedRecipe> consumer) {
 		boil(consumer, teapot_water, teapot_hot);
 		boil(consumer, teapot_milk, teapot_frothed);
 
+		// empty teapots
+		ShapelessRecipeBuilder.shapeless(RecipeCategory.MISC, Items.WATER_BUCKET)
+											 .requires(teapot_water)
+											 .requires(Items.BUCKET)
+											 .unlockedBy("has_item", has(teapot_water))
+											 .save(consumer, "water_bucket_from_teapot");
```

Comment:
Recipes should never go under `minecraft`. they should be under `simplytea`. If there is no resource location parameter set the namespace as a string, `simplytea:water_bucket_from_teapot`

Reply:
All recipes are going into the simplytea namespace now.